File: soap_api.py
import zeep

# WSDL URL (defines the API contract)
wsdl = "http://www.dneonline.com/calculator.asmx?WSDL"

client = zeep.Client(wsdl=wsdl)

# Call the SOAP method
result = client.service.Add(intA=5, intB=3)

print(result)  # Output: 8

File: soap_api.js
const soap = require('soap');

const url = "http://www.dneonline.com/calculator.asmx?WSDL";

soap.createClient(url, function(err, client) {
  if (err) throw err;

  client.Add({ intA: 5, intB: 3 }, function(err, result) {
    if (err) throw err;

    console.log(result.AddResult); // Output: 8
  });
});